tests/testthat/test-resource.R

```diff
@@ -0,0 +1,48 @@
+test_that("Validate resource args", {
+  expect_silent(validate_cores(Inf))
+  expect_error(validate_cores(-Inf))
+  expect_silent(validate_cores(1))
+  expect_error(validate_cores(-1))
+  expect_error(validate_cores("potato"))
+  expect_error(validate_cores(NULL))
+  expect_error(validate_cores(NA))
+  expect_error(validate_cores(mtcars))
+  expect_error(validate_cores(c(1, 2, 3)))
+
+  expect_error(validate_runtime(c(1, 2, 3)))
```

Comment:
suggest breaking these into different test_that blocks at exactly the point you have newlines